recipe/meta.yaml

```diff
@@ -22,6 +21,7 @@ requirements:
     - python >=3.7
     - python-lsp-server
     - ruff >=0.0.260
+    - tomli >=1.1.0  # [py<311]
```

Comment:
while this will "punish" py 3.11 users with a 15kb download, in almost all cases, `noarch: python` should be preserved basically at all cost, otherwise we have to do `python * os * architecture` builds. I recommend restoring `noarch: python` and removing this selector.

Reply:
Done :+1: